package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@ebay/retriever",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
If you don't mind, I'd prefer to do the version change alone in a separate commit after merge